apps/examples/fs_helper/fs_rw_test.c

```diff
@@ -0,0 +1,271 @@
+/****************************************************************************
+ *
+ * Copyright 2018 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <tinyara/config.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/time.h>
+#include <fcntl.h>
+#include <errno.h>
+
+/****************************************************************************
+ * Definitions
+ ****************************************************************************/
+#define USAGE                                                           \
+    "\n FS R/W Performance Test\n"                                      \
+    "\n usage: fs_rw_test Path Size Iteration Operation Detail\n"       \
+    "\n [Path] : Test File Path\n"                                      \
+    " [Size] : Data size\n"                                             \
+    " [Iteration] : Number of repetitions\n"                            \
+    " [Operation] : 1.Read/Write / 2.Read / 3.Write\n"                  \
+    " [Detail] : Details Print Option [y/n]\n"                          \
```

Comment:
Option of printing details? Option to print details?

Reply:
or show details?

Reply:
print is a view of app, but USAGE is for user. So, `show` is better to understand to user.

Reply:
I made it easier to understand.